### ISSUE TITLE
Compositional profile of mucosal bacteriome of smokers and smokeless tobacco users

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/BugSigDBcuration#56]
https://pubmed.ncbi.nlm.nih.gov/34436669/

**Comment by Cindy-erella** [comment]
Kindly assign this to me. Cindy

**Comment by cmirzayi** [comment]
Hi @Cindy-erella  Thank you so much for your contributions to the BugSigDB project. I wanted to check to see if you wanted to finish this curation. I understand that with the Outreachy applicant selection process over, you may no longer wish to help out with BugSigDB or have other priorities right now.

If not, I will open it up to other curators to finish the curation.

**Comment by Cindy-erella** [comment]
Hello @cmirzayi thank-you for reaching out. I will finish this curation in due time. Thankyou for the reminder

**Comment by atrayees** [comment]
Curated at https://bugsigdb.org/Study_466

**Comment by lwaldron** [comment]
Could you post here the PDF for this? I'm actually having trouble accessing it now :D 

One quick thing I noticed though @atrayees  - when the study has a PMID, you don't need to fill in DOI or URI (in this case the URI is redundant with the PMID, and the DOI is a dead link because the wiki adds a DOI prefix)

**Comment by atrayees** [comment]
[s00784-021-04137-7.pdf](https://github.com/waldronlab/BugSigDBcuration/files/11737430/s00784-021-04137-7.pdf)
 
Yes, sure! And about the the PMID thing, thank you for telling me, I'll keep that in mind

**Comment by lwaldron** [comment]
* This is a case-control design because the observations at a single point in time, not a prospective cohort (which involves following a cohort over time, see https://www.cancer.gov/publications/dictionaries/cancer-terms/def/prospective-cohort-study)
* Experiment 1, "chewing tobacco behavior" is more accurate than "smoking behavior"
* Experiment 1, I moved "sex" from "Confounders controlled for" to "Matched on" since cases and controls were matched for sex (by choosing all males), not otherwise controlled for during analysis (such as by using a multivariate regression). 
* Experiment 1 / Signature 1: I think the source is Figure 5, not 3. Also, it listed class _Fusobacteriia_ but the paper listed genus _Fusobacterium_.  
*  Experiment 2 / Signature 1: _Candidatus Saccharibacteria_ is a phylum so it's unlikely what they were referring to by _Saccharibacteria_ since they were probably referring to a genus. This one isn't very clear to me but I like better [unclassified Candidatus Saccharibacteria](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Tree&id=1895827&lvl=3&p=has_linkout&p=blast_url&p=genome_blast&keep=1&srchmode=1&unlock) (Taxonomy ID: 1895827, no rank)
* Experiment 3: I think _Fusobacterium_ and _Campylobacter_ are *increased* in tobacco chewers and _decreased_ in smokers. I reversed your group 0 and group 1 since smokers are more common as a group 1 and that way I left "Smoking behavior" as the condition (when "Smoking behavior" is the condition as it was, I expect to see smokers as group 1).
